README.md

```diff
@@ -891,7 +894,43 @@ Now the last required step is to register the `MusicianSerializer` to the config
 
 From now on, Hazelcast will use `MusicianSerializer` to serialize `Musician` objects.
 
-## 4.4. Global Serialization
+## 4.4. JSON Serialization
+
+You can use JSON formatted strings as objects in Hazelcast cluster. Starting with Hazelcast server version 3.12, the JSON serialization is one of the formerly supported serialization methods. Creating JSON objects in the cluster does not require any server side coding and hence you can just send a JSOn formatted string object to the cluster and query these objects by fields.
```

Comment:
...use the JSON formatted strings as objects in the Hazelcast cluster. ...

... Starting with Hazelcast IMDG 3.12, the JSON serialization is one of the supported serialization methods ...

... you can just send a JSON ...